thunderstorm/utils.py

```diff
@@ -86,3 +91,75 @@ def paginate(query, page, page_size, url_path):
     pagination_info = get_pagination_info(page, page_size, num_records, url_path)
 
     return query.offset(start).limit(page_size), pagination_info
+
+
+def make_paginated_response(query, url_path, schema, page, page_size):
+    """
+    Take a sqlalchemy query and paginate it based on the args provided.
+
+    Args:
+        query (sqlalchemy Query object): Query you wish to paginate
+        url_path (str): url_path that the request was sent to e.g. for the url
+            'http://localhost/foo/bar/baz/' it would be '/foo/bar/baz'.
+            Query parameters are stripped out by the get_pagination_info func
+        schema (Schema): Subclass of marshmallow.Schema
+        page (int): Page number of the results page to return
+        page_size (int): Number of results to return per page
+
+    Returns:
+        dict: The structure of which conforms to the thunderstorm API
+            spec response structure
+    """
+    start = (page - 1) * page_size
+    num_records = query.count()
+
+    response = get_pagination_info(page, page_size, num_records, url_path)
+    query = query.offset(start).limit(page_size)
+
+    # python34 does not support **pagination_info when creating a dict so update the repsonse dict manually
```

Comment:
```suggestion
    # python34 does not support **pagination_info when creating a dict so update the response dict manually
```

also we should dump 3.4 support

Reply:
done